python/src/opendp/extras/polars/__init__.py

```diff
@@ -321,11 +320,14 @@ def lazy(self):
 
         Requires "honest-but-curious" because the privacy guarantees only apply if:
         1. The LazyFrame (compute plan) is only ever executed once.
-        2. The analyst does not observe ordering of rows in the output. To ensure this, shuffle the output.
+        2. The analyst does not observe ordering of rows in the output. 
+        
+        To ensure that row ordering is not observed:
+        1. Do not extend the compute plan with order-sensitive computations.
+        2. Shuffle the output once collected.
         """
         from opendp._data import onceframe_lazy
 
-        assert_features("honest-but-curious")
```

Comment:
Shouldn't this still be required? Or am I missing something?

Reply:
It's already asserted on onceframe_lazy

Reply:
I'd have a slight preference to keep it. The cost is negligible, and if some moderately paranoid person is reviewing the code in the future, it will save them them trouble of confirming that the wrapped function has the assert.